packages/client/mutations/AcceptTeamInvitationMutation.ts

```diff
@@ -143,6 +144,7 @@ export const handleAcceptTeamInvitationErrors = (
 ) => {
   if (acceptTeamInvitation?.error) {
     const {message} = acceptTeamInvitation.error
+    if (message === InvitationTokenError.ALREADY_ACCEPTED) return true
```

Comment:
I think we shouldn't show the `accepted` snackbar to users after they login, or change the snack message to be more user friendly